### PR TITLE
feat: drgx driver

### LINKED ISSUE #117: Integration: DRGx
**Exchange name**
DRGx

**Website**
https://www.drgx.io

**API endpoint**
https://feeds-api.drgx.io/drgx/tickers.json

**API documentation**

- Exchange Swagger API: https://exchange.drgx.io/api#/
- Trades API: https://exchange.drgx.io/cmc/public/TradesCm?pair=eth_btc
- OrderBook API: https://exchange.drgx.io/cmc/public/orderbookCm?pair=drg_eth
- Ticker API 1: https://feeds-api.drgx.io/drgx/tickers.json
- Ticker API 2: https://exchange.drgx.io/cmc/public/returnTicker

### PR DESCRIPTION
**Pull request by abdul**
Fixes #117

### CODE REVIEW

File: drivers/drgx.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    const ticker = tickers[pair];
+
+    return new Ticker({
+      base,
+      quote,
+      quoteVolume: parseToFloat(ticker.quoteVolume24hr),
+      baseVolume: parseToFloat(ticker.volume24hr),
+      close: parseToFloat(ticker.last),
+      high: parseToFloat(ticker.high24hr),
+      low: parseToFloat(ticker.low24hr),
```
Comment (sajcics) [inline drivers/drgx.js]: @abdul I will also add ask and bid price because it's available in API :)

Comment (abdul) [reply drivers/drgx.js]: @sajcics Thank you, I am going to update the pull request.

Comment (abdul) [reply drivers/drgx.js]: @sajcics I have updated the code, can you please review it again?

**Pull request merged**